backstop/scripts/ready.js

```diff
@@ -113,6 +113,33 @@ module.exports = async function (page, scenario, vp) {
     await page.waitForSelector('.cbFormErrorMarker', {visible: true})
   }
 
+  let leafletMapInitialized = await page.evaluate(async function () {
+    let initialized = undefined;
+    var containers = document.querySelectorAll(".js-leaflet-map");
+    if (containers.length) {
+      containers.forEach(function (e) {
+        var container = L.DomUtil.get(e);
+        if (container != null) {
```

Comment:
can you add some notes to these conditions?  

I feel like the condition should be reserved - currently, `if (containers.length)` is false, then `initialized` is set to true - that means for pages that don't have leaflet maps, we are adding a 5 second timeout ?

Reply:
We are not running this on the pages that don't have a map.